compliance/collection/main.go

```diff
@@ -181,6 +183,47 @@ func initializeStream(ctx context.Context, cli sensor.ComplianceServiceClient) (
 	return client, config, nil
 }
 
+func manageNodescanLoop(ctx context.Context, cli sensor.ComplianceServiceClient) {
+	client, _, err := initializeStream(ctx, cli)
+	if err != nil {
+		log.Fatalf("error initializing stream to sensor: %v", err)
+	}
+	t := time.NewTicker(env.NodeScanInterval.DurationSetting())
+	scanner := nodescanv2.FakeNodeScanner{} // FIXME: Replace with real scanner (ROX-12971)
```

Comment:
This is our main dependency of that stream-handling loop - maybe we should inject `scanner` from the main function?